cocos/2d/components/rich-text.ts

```diff
@@ -582,14 +582,14 @@ export class RichText extends Component {
             return partStringArr;
         }
 
-        const labelSize = this._calculateSize(styleIndex, text);
+        const labelSize = this._calculateSize(styleIndex, text).clone();
```

Comment:
Why needs a new vec2?

Reply:
Because the function returns an object, it is not safe to use it directly